src/System.Runtime/src/ApiCompatBaseline.uap101aot.txt

```diff
@@ -485,4 +485,6 @@ MembersMustExist : Member 'System.IO.FileLoadException.FusionLog.get()' does not
 MembersMustExist : Member 'System.IO.FileNotFoundException.FusionLog.get()' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.IO.Stream.CreateWaitHandle()' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.IO.Stream.ObjectInvariant()' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.IO.Stream.Synchronized(System.IO.Stream)' does not exist in the implementation but it does exist in the contract.
\ No newline at end of file
+MembersMustExist : Member 'System.IO.Stream.Synchronized(System.IO.Stream)' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'System.MulticastNotSupportedException' does not exist in the implementation but it does exist in the contract.
```

Comment:
good point, I'll do that next time. For now I'll just refresh this whole file so that everything gets reordered correctly and all of the duplicates are gone.